tests/dummy/app/models/fake-model.js

```diff
@@ -54,6 +55,10 @@ export default DS.Model.extend(Validator,{
     lotteryNumber: {
       numericality: true
     },
+    acceptConditions: {
+      acceptance: { acceptableValues: [true] }
```

Comment:
@dmuneras Could you change the option `acceptableValues` to `accept`. And use a string like 'yes' to test it.